simulations/ContestSimulation.java

```diff
@@ -0,0 +1,192 @@
+        .exec(publicScoreboard) // Fetch the public scoreboard
+        .pause(1,5)               // Brief delay
+        .exec(login())            // Log in as the team
+
+        // Mimic the behavior of the team page, it autorefreshes every 30s
+        .exec(
+            poll().pollerName("teampage")
+            .every(30)
+            .exec(teamPage)
+        )
+
+        // While we're polling the teampage, lets pretend to do some actions like a real team might
+        .repeat(NUM_TEAM_ACTIONS, "i").on(
+            exec(
+                randomSwitch().on(
+                    // 20% chance of making a submission
+                    Choice.withWeight(20.0, exec(
+                        uniformRandomSwitch().on(
+                            exec(teamSubmitC),
+                            exec(teamSubmitCpp),
+                            exec(teamSubmitJava),
+                            exec(teamSubmitKotlin),
+                            exec(teamSubmitPython)
+                        )
+                    )),
+                    // 10% of requesting clarification
+                    Choice.withWeight(10.0, teamRequestClarification),
+                    // 30% of loading problems page
+                    Choice.withWeight(30.0, teamProblems),
+                    // 40% of loading scoreboard
+                    Choice.withWeight(40.0, teamScoreboard)
+                )
+            )
+            // Sleep anywhere between 5s and 1 minute before doing another action
+            .pause(5,60)
+        )
+
+        // Stop polling the teampage
+        .exec(
+            poll().pollerName("teampage").stop()
+        )
+
+    ;
+
+
+    HttpProtocolBuilder httpProtocol = http
+        .baseUrl(System.getProperty("baseurl"))
+        .inferHtmlResources()   // load all dependent things like images/js/css/etc
+        .nameInferredHtmlResourcesAfterPath()
+    ;
+
+    {
+        setUp(
+            // teamScenario.injectOpen(atOnceUsers(5)),
+            teamScenario.injectClosed(
+                incrementConcurrentUsers(10)  // Batches of this many users
+                .times(10)                    // how many "levels" to test
+                .eachLevelLasting(60)         // hold the number of users for this duration
+                .separatedByRampsLasting(60)  // Take 60s between adding each user
+                .startingFrom(10)             // Start with 10 users
+            ),
+            // spectatorScenario.injectOpen(atOnceUsers(10))
+            spectatorScenario.injectClosed(
+                incrementConcurrentUsers(10)  // Batches of this many spectators
+                .times(10)                    // how many "levels" to test
+                .eachLevelLasting(60)         // hold the number of users for this duration
+                .separatedByRampsLasting(60)  // Take 60s between adding each user
+                .startingFrom(10)             // Start with 10 users
+            )
```

Comment:
This is the bit you'll want to adjust to tune how many concurrent users are hammering the system. Probably play with these things until it falls over.